tooling/test/test.go

```diff
@@ -14,12 +14,17 @@ type SugarTest struct {
 	Hint      string
 	Request   RequestBuilder
 	Requests  []RequestBuilder
-	Response  ExpectBuilder
+	Response  ExpectValidator
 	Responses ExpectsBuilder
 }
 
 type SugarTests []SugarTest
 
+func (s *SugarTests) Append(tests ...SugarTest) *SugarTests {
+	*s = append(*s, tests...)
+	return s
+}
```

Comment:
Added this helper since the pattern with struct literals we were using didn't work with helpers that returned multiple tests.

WDYT?

Reply:
SGTM, anything stops us from using a Value Received instead of a pointer here?

That'll make the test code more consistent and remove a side-effect.

Reply:
@hacdias do you have more details? Else I'd rather keep a Value here